unstructured/partition/lang.py

```diff
@@ -200,61 +200,70 @@ def convert_old_ocr_languages_to_languages(ocr_languages: str):
     """
-    Convert a language code to its tesseract formatted and recognized langcode(s), if supported.
+    Convert a single language code to its tesseract formatted and recognized
+    langcode(s), if supported.
     """
     # if language is already tesseract langcode, return it immediately
     # this will catch the tesseract special cases equ and osd
     # NOTE(shreya): this may catch some cases of choosing between tesseract code variants for a lang
-    if lang in PYTESSERACT_LANGS:
+    if lang in PYTESSERACT_LANG_CODES:
         return lang
 
-    # get iso639 language object
-    try:
-        lang_iso639 = iso639.Language.match(lang.lower())
-    except iso639.LanguageNotFoundError:
-        logger.warning(f"{lang} is not a valid standard language code.")
-        return ""
+    lang_iso639 = _get_iso639_language_object(lang)
 
     # tesseract uses 3 digit codes (639-3, 639-2b, etc) as prefixes, with suffixes for orthography
     # use first 3 letters of tesseract codes for matching to standard codes
-    pytesseract_langs_3 = {lang[:3] for lang in PYTESSERACT_LANGS}
+    pytesseract_langs_3 = {lang[:3] for lang in PYTESSERACT_LANG_CODES}
 
-    # try to match ISO 639-3 code
-    if lang_iso639.part3 in pytesseract_langs_3:
-        matched_langcodes = _get_all_tesseract_langcodes_with_prefix(lang_iso639.part3)
-        return TESSERACT_LANGUAGES_SPLITTER.join(matched_langcodes)
+    if lang_iso639:
+        # try to match ISO 639-3 code
+        if lang_iso639.part3 in pytesseract_langs_3:
+            matched_langcodes = _get_all_tesseract_langcodes_with_prefix(lang_iso639.part3)
+            return TESSERACT_LANGUAGES_SPLITTER.join(matched_langcodes)
 
-    # try to match ISO 639-2b
-    elif lang_iso639.part2b in pytesseract_langs_3:
-        matched_langcodes = _get_all_tesseract_langcodes_with_prefix(lang_iso639.part2b)
-        return TESSERACT_LANGUAGES_SPLITTER.join(matched_langcodes)
+        # try to match ISO 639-2b
+        elif lang_iso639.part2b in pytesseract_langs_3:
+            matched_langcodes = _get_all_tesseract_langcodes_with_prefix(lang_iso639.part2b)
+            return TESSERACT_LANGUAGES_SPLITTER.join(matched_langcodes)
 
-    # try to match ISO 639-2t
-    elif lang_iso639.part2t in pytesseract_langs_3:
-        matched_langcodes = _get_all_tesseract_langcodes_with_prefix(lang_iso639.part2t)
-        return TESSERACT_LANGUAGES_SPLITTER.join(matched_langcodes)
+        # try to match ISO 639-2t
+        elif lang_iso639.part2t in pytesseract_langs_3:
+            matched_langcodes = _get_all_tesseract_langcodes_with_prefix(lang_iso639.part2t)
+            return TESSERACT_LANGUAGES_SPLITTER.join(matched_langcodes)
 
-    else:
-        logger.warning(f"{lang} is not a language supported by Tesseract.")
-        return ""
+        else:
+            logger.warning(f"{lang} is not a language supported by Tesseract.")
+            return ""
+    return ""
```

Comment:
If line 220 (`if lang_iso639:`) fails it comes to this return but doesn't provide a warning as to what happened. Can we include the warning from line 237 (`logger.warning(f"{lang} is not a language supported by Tesseract.")`) or something similar for this case as well?